example-databinding/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
+        android:name=".ExempleApplication"
```

Comment:
typo :)

Reply:
I move Application into shared at the same time, to enable Leaks canary into the other example